src/server.ts

```diff
@@ -0,0 +1,40 @@
+import 'reflect-metadata'; // this shim is required
+import { createConnection, Connection } from 'typeorm';
+import AdminBro from 'admin-bro';
+//TODO: Check how to change this to TS import
+const AdminBroExpress = require('@admin-bro/express');
+import { Database, Resource } from '@admin-bro/typeorm';
+import { User } from './entities/user.entity';
+import { PORT } from './config';
+import connection from 'database/connection';
+import app from 'app';
+
+const handleConnection = async (connection: Connection) => {
+  // creates admin panel, register all resources and their options
+  AdminBro.registerAdapter({ Database, Resource });
+  const adminBro = new AdminBro({
+    databases: [connection],
+    resources: [
+      {
+        resource: User,
+        options: { parent: { name: 'resources', icon: 'Home' } }
+      }
+    ],
+    rootPath: '/admin',
```

Comment:
Maybe create a paths file and import from there, I think it will be more scalable in the medium/long term

Reply:
🤔 This would be more like a hardcoded path, since we're not planning to have more custom routes, just the ones for the controllers, and those are defined inside each controller by itself.

Reply:
Oh I didn't know that, If that is the case is fine to hardcode this path, thanks for the comment @Alxzu 🙌